src/libs/actions/CloseAccount.js

```diff
@@ -1,33 +1,14 @@
 import Onyx from 'react-native-onyx';
 import ONYXKEYS from '../../ONYXKEYS';
 
-let isCloseAccountModalOpen;
-Onyx.connect({
-    key: ONYXKEYS.IS_CLOSE_ACCOUNT_MODAL_OPEN,
-    callback: flag => isCloseAccountModalOpen = flag,
-});
-
-/**
- * Set CloseAccount flag to show modal
- */
-function showCloseAccountModal() {
-    if (isCloseAccountModalOpen) {
-        return;
-    }
-    Onyx.set(ONYXKEYS.IS_CLOSE_ACCOUNT_MODAL_OPEN, true);
-}
-
 /**
-* Unset CloseAccount flag to hide modal
+ * Clear CloseAccount error message to hide modal
  */
-function hideCloseAccountModal() {
-    if (!isCloseAccountModalOpen) {
-        return;
-    }
-    Onyx.set(ONYXKEYS.IS_CLOSE_ACCOUNT_MODAL_OPEN, false);
+function clearError() {
+    Onyx.merge(ONYXKEYS.CLOSE_ACCOUNT, {error: ''});
 }
 
 export {
-    showCloseAccountModal,
-    hideCloseAccountModal,
+    // eslint-disable-next-line import/prefer-default-export
+    clearError,
 };
```

Comment:
Seems like we don't use this in too many places, but the use does make sense given it's an action file.

Reply:
Yeahhh if I remember correctly, our linter got mad when I tried to do a default export in an `action` file?

Reply:
No no that doesn't make sense, I see that all over

Reply:
Oh oh i saw it - Linting prefered a default export when there's only 1 function to export, but doing something like `import CloseAccountClearError from '...';` felt weird, so I kept it this way

Reply:
Yeah exactly. This seems much better